layers/gpuav/shaders/gpuav_error_codes.h

```diff
@@ -144,6 +145,11 @@ const int kErrorSubCodePreCopyMemoryIndirectSrcAddressInvalid = 7;
 const int kErrorSubCodePreCopyMemoryIndirectDstAddressInvalid = 8;
 const int kErrorSubCodePreCopyMemoryToImageIndirectSrcAddressInvalid = 9;
 
+// Pre Build Acceleration Structures
+//
+const int kErrorSubCodePreBuildAccelerationStructures_InvalidAS = 1;
```

Comment:
Just want to say in the future, I would be happy to "namespace" (for GLSL sake) these things with underscores like

```suggestion
const int kErrorSubCode_PreBuildAccelerationStructures_InvalidAS = 1;
```

so what you have is good, saying we should do it for others later too